### ISSUE TITLE
Cornelius' Gold Watch, dynamite does not work

### ISSUE THREAD

**Issue by bruenor41** [alexbatalov/fallout2-ce#253]
Hi,

I am in Modoc and I want finish the quest Cornelius' Gold Watch. I place dynamite under latrine. After placing it I climb up and I get message that I was hit for no damage, but nothing else happens.

I use build for android

**Comment by alexbatalov** [comment]
Can you provide a savegame?

**Comment by bruenor41** [comment]
https://www.dropbox.com/s/t0zb189bmu82o0u/SAVEGAME_cz.zip?dl=0

Save3 is before entering latrine. Save4 is another bug. I try to rape Mira, but then everything became black

**Comment by alexbatalov** [comment]
I've checked dynamite stuff, it looks good, at least on Windows x64. If you place dynamite too far from the rocks, nothing happens (vanilla behaviour). When you place it near the rocks, the cabin is destroyed with explosion, leaving shitty piles all over the place. At this time I cannot check on Android, but will try tomorrow. Please check where you place dynamite.

As for Miria, you're right, there was a bug in dropping all items routine. Miria and Grisham have very intertwined scripting with lots of context switching. Miria is the one who fade screen out. Grisham's script expected PC to have no items in order to fade screen in. Since not all items where dropped, fading in never happened.

**Comment by bruenor41** [comment]
Hi, thank you very much for quick response. I placed dynamite on various places. If I drop it and climb up, then I see message informing me that I was hurt for no damage. Seems it exploded. Then I climb down and I search for dynamite, but is not there. So I think it really exploded, but it did nothing.

**Comment by alexbatalov** [comment]
I've checked your savegame on Android, it works as expected - place dynamite next to the rocks, the timer does not matter (I've used 1 minute), climb up. The cabin is destroyed with explosion. Go down to the big hole. This time the passage is cleared of rocks, go north-west, kill mole rat, grab watch.

Here is the place for dynamite:
![dynamite](https://user-images.githubusercontent.com/172978/222081778-0f2df6e7-88cd-4295-9f3e-17ace6b18377.png)

Let me know if something doesn't work for you.
